src/Aspire.Hosting/Dcp/ResourceSnapshotBuilder.cs

```diff
@@ -147,6 +158,31 @@ public CustomResourceSnapshot ToSnapshot(Executable executable, CustomResourceSn
         };
     }
 
+    private static (ImmutableArray<string> Args, ImmutableArray<int>? ArgsAreSensitive, bool IsSensitive)? GetLaunchArgs(CustomResource resource)
```

Comment:
Is there a particular reason why the sensitivity flags are an `int` array and not `bool` array?

Reply:
Not sure if it's the protobuf library or gRPC, but an ImmutableArray<bool> was not able to be serialized/deserialized, and instead was being treated as a string. using int was a workaround